dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/JaxWsClientProxy.java

```diff
@@ -160,21 +167,33 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
         }
         return adjustObject(result);
     }
+    
     Exception mapException(Method method, BindingOperationInfo boi, Exception ex) {
+        
+        boolean isFinestEnabled = LOG.isLoggable(Level.FINEST);   // Liberty Change issue #26529
         if (method != null) {
             for (Class<?> excls : method.getExceptionTypes()) {
                 if (excls.isInstance(ex)) {
+                    if(isFinestEnabled)  {
```

Comment:
For consistency with trace in other modules, do we need to mark the Liberty change for the complete if statement?